Dockerfile

```diff
@@ -1,12 +1,13 @@
 FROM danlynn/ember-cli:latest as build
 
-COPY package.json yarn.lock /myapp/
+COPY package.json pnpm-lock.yaml /myapp/
 
-RUN yarn install
+RUN npm install -g pnpm
```

Comment:
Put this above the `COPY` command. Currently, this layer will be re-run if there are changes in `package.json` or `pnpm-lock.yaml`. However, this is not necessary because those files have no effect on this command.